native/src/main/scala/sqs4s/api/lo/ReceiveMessage.scala

```diff
@@ -0,0 +1,88 @@
+package sqs4s.api.lo
+
+import cats.effect.{Clock, Sync}
+import cats.implicits._
+import org.http4s.client.Client
+import org.http4s.scalaxml._
+import org.http4s.{Method, Request, Uri}
+import sqs4s.api.SqsSetting
+import sqs4s.api.errors.SqsError
+import sqs4s.internal.aws4.common._
+import sqs4s.internal.models.CReq
+import sqs4s.serialization.MessageDecoder
+
+import scala.xml.{Elem, XML}
+
+case class ReceiveMessage[F[_]: Sync: Clock, T](
+  queue: Uri,
+  maxNumberOfMessages: Int = 1,
+  visibilityTimeout: Int = 15,
+  attributes: Map[String, String] = Map.empty,
+  waitTimeSeconds: Option[Int] = None
+)(implicit decoder: MessageDecoder[F, String, String, T])
+    extends Action[F, Seq[ReceiveMessage.Result[T]]] {
+
+  def runWith(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)